webapp/components/login/login_controller.jsx

```diff
@@ -387,10 +390,16 @@ export default class LoginController extends React.Component {
                                 type='submit'
                                 className='btn btn-primary'
                             >
-                                <FormattedMessage
+                                {this.state.loading ? <span>
+                                    <span className='fa fa-refresh icon--rotate'/>
+                                    <FormattedMessage
+                                        id='login.signInLoading'
+                                        defaultMessage='Signing in...'
+                                    />
+                                </span> : <FormattedMessage
```

Comment:
Minor, but could you move this to use a regular if statement? We try to avoid using the tertiary operator as it makes the code less readable